internal/controller/drplacementcontrol.go

```diff
@@ -821,7 +825,7 @@ func checkActivationForStorageIdentifier(
 //   - Check if current primary (that is not the preferred cluster), is ready to switch over
 //   - Relocate!
 //
-//nolint:gocognit,cyclop,funlen
+//nolint:gocognit,gocyclo,cyclop,funlen
```

Comment:
We need to refactor this code so we can make the change without skipping more linters.